standard/expressions.md

```diff
@@ -3557,6 +3643,47 @@ The result of evaluating `~x`, where `X` is an expression of an enumeration ty
 
 Lifted ([§12.4.8](expressions.md#1248-lifted-operators)) forms of the unlifted predefined bitwise complement operators defined above are also predefined.
 
+### §index-from-end-operator Index-from-end operator
```

Comment:
Somewhere in this section (perhaps in a note), we should state that `^0` represents the end of the sequence, not the last element. `^1` represents the last element. (Yes, I know it's in a comment in the example, but still...)

Reply:
I think that should be explained where `System.Index` is introduced or here, with x-refs between them.